components/admin_console/team_channel_settings/channel/details/channel_details.tsx

```diff
@@ -468,7 +468,7 @@ export default class ChannelDetails extends React.PureComponent<ChannelDetailsPr
                     trackEvent('admin_channel_config_page', 'groups_added_to_channel', {count: link.length, channel_id: channelID});
                 }
 
-                const actionsToAwait: any[] = [actions.getGroups(channelID)];
+                const actionsToAwait: any[] = [actions.getGroups(channelID, undefined, undefined, undefined, false)];
```

Comment:
Is this function not letting you ignore the last argument as optional? It looks like the function definition itself might be wrong.

Reply:
I have a `TS2554: Expected 5 arguments, but got 4.` if I don't provide it. Method signature is from `getGroupsAssociatedToChannel` from mattermost-redux. Not sure about how to fix it. Any idea?

Reply:
We'll need to fix the type definition in `mattermost-redux`. I'll make a fix.

Reply:
https://github.com/mattermost/mattermost-redux/pull/1281

Reply:
@cinlloc Above PR has been merged so you should be able to resolve this issue.

Reply:
ok, fixed in 24e2c48.